src/plugins/placeholder.ts

```diff
@@ -117,10 +117,23 @@ export default () =>
               const $pos = state.doc.resolve(pos)
 
               let placeholderText = 'Type heading here'
-              if (
-                findParentNodeOfTypeClosestToPos($pos, schema.nodes.box_element)
-              ) {
-                placeholderText = 'Optional box title...'
+
+              const boxElement = findParentNodeOfTypeClosestToPos(
+                $pos,
+                schema.nodes.box_element
+              )
+
+              if (boxElement) {
+                const section = findParentNodeOfTypeClosestToPos(
+                  $pos,
+                  schema.nodes.section
+                )
+
+                // If the section is a direct child of the box_element,
+                // it's the top-level section and should have the box placeholder
+                if (section && section.depth === boxElement.depth + 1) {
+                  placeholderText = 'Optional box title...'
+                }
```

Comment:
if it's for me I will use `findChildrenByType(boxElement.node, schema.nodes.section)` instead of checking depth